cwms-data-api/src/main/java/cwms/cda/api/RatingController.java

```diff
@@ -134,24 +139,18 @@ private ContentType getContentType(Context ctx) {
         return contentType;
     }
 
-    private RatingSet deserializeRatingSet(Context ctx) throws IOException, RatingException {
-        return deserializeRatingSet(ctx.body(), getContentType(ctx));
+    private String deserializeRatingSet(Context ctx) throws IOException, RatingException {
+        return deserializeRatingSet(ctx.body(), getContentType(ctx).getType());
     }
 
-
-    public RatingSet deserializeRatingSet(String body, ContentType contentType)
-            throws IOException, RatingException {
-        return deserializeRatingSet(body, contentType.getType());
-    }
-
-    public RatingSet deserializeRatingSet(String body, String contentType) throws IOException,
+    String deserializeRatingSet(String body, String contentType) throws IOException,
```

Comment:
Shouldn't this be renamed to "serializeRatingSet"?
Seems like the operation was inverted.

Reply:
I'm not going to tweak the name. I think some of the confusion stems from how the libraries we use work.

Plus this just needs to get into `develop`, I was a little surprised to see it in this branch but it is needed for the Rating tests to work.